README.md

```diff
@@ -1,7 +1,7 @@
 # SYSC4806-TeamJORA
 
 # Project Description
-Make an application that will allow students to evaluate their peers.  An Instructor will set up a list of items to be assessed, with a rubric for each item. Students are part of one and only one group. Students can add themselves to an existing group or create a new one. The Instructor can also create groups and assign existing students to a group. The instructor can also populate group memberships by uploading a file. A student can assess the peers in his/her group as per the rubric. The Instructor can also generate a report which has the average assessment of each student (average of assessments done about (not by) this student) for each item. The report should be downloadable in a CSV file so that the instructor can further process it in a spreadsheet.
+Make an application that will allow students to evaluate their peers.  An Model.Instructor will set up a list of items to be assessed, with a rubric for each item. Students are part of one and only one group. Students can add themselves to an existing group or create a new one. The Model.Instructor can also create groups and assign existing students to a group. The instructor can also populate group memberships by uploading a file. A student can assess the peers in his/her group as per the rubric. The Model.Instructor can also generate a report which has the average assessment of each student (average of assessments done about (not by) this student) for each item. The report should be downloadable in a CSV file so that the instructor can further process it in a spreadsheet.
 
 # Package Description
   Application: For back-end processes\
```

Comment:
You should add the start up info you told us about into the read me under # Start Up

Reply:
Thanks for reminding me